portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,34 +30,37 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  private List<String> comments;
-
-  @Override
-  public void init() {
-    comments = new ArrayList<Object>();
-  }
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
 
+    int numComments = getNumComments(request);
+
+    // decreased by 1 in order to count starting from 1 instead of 0
+    numComments--;
```

Comment:
I would recommend keeping numComments unchanged (otherwise the name does not match the value), and change the count > numComments below to count >= numComments.

Reply:
Okay, thanks!